mbta.js

```diff
@@ -0,0 +1,36 @@
+const subway ={
+    Red: ['South Station'
+    , 'Park Street',
+     'Kendall', 
+     'Central', 
+    'Harvard', 
+    'Porter', 
+    'Davis', 
+    'Alewife'],
+   Green:  ['Government Center', 
+   'Park Street',
+    'Boylston', 
+    'Arlington',
+     'Copley',
+      'Hynes', 
+      'Kenmore'],
+   Orange: ['North Station','Haymarket', 
+    'Park Street', 
+    'State',
+     'Downtown Crossing',
+      'Chinatown','Back Bay', 'Forest Hills']
+   }
+   const stopsBetweenStations = function (startLine, startStation, endLine, endStation) {
+  const indexStartStation = subway[startLine].indexOf(startStation)
+  const indexEndStation = subway[endLine].indexOf(endStation)
+  if (startLine === endLine) {
+    return Math.abs(indexEndStation - indexStartStation)
+  }
+    const indexParkStreetSl = subway[startLine].indexOf('Park Street');
+    const indexParkStreetEl = subway[endLine].indexOf('Park Steet');
+    const startLineDist = Math.abs(indexParkStreetSl - indexStartStation);
+    const endlineDist = Math.abs(indexParkStreetEl - indexEndStation);
+  const totel=startLineDist+endlineDist;
+    return totel;
```

Comment:
your end result if the user changes stations is always off by 2.
I did these two tests:
stopsBetweenStations('Red', 'Park street', 'Red', 'South Station')
1
stopsBetweenStations('Green', 'Park street', 'Red', 'South Station')
3
even though Park street in any line should be the same. please fix this issue.